app/models/account.rb

```diff
@@ -415,22 +417,12 @@ def verifiable?
     end
 
     def mark_verified!
-      @verified_at = Time.now.utc
-      @attributes['verified_at'] = @verified_at
+      self.verified_at = Time.now.utc
+      @original_field['verified_at'] = @verified_at
```

Comment:
I had to maintain this process to propagate the changes to the fields of account.